### ISSUE TITLE
OAuth2TokenResponse.reflesh_token should be optional

### ISSUE THREAD

**Issue by fuyutsuki** [OpenXbox/xbox-webapi-python#36]
If you remove `Xboxlive.offline_access` from the scope, the refresh_token is set to `None` and the following error occurs.

```bash
pydantic.error_wrappers.ValidationError: 1 validation error for OAuth2TokenResponse
refresh_token
  field required (type=value_error.missing)
```

Hence, I believe that refresh_token should be set to Optional[str].

**Comment by tuxuser** [comment]
Thanks for the report.

**Comment by tuxuser** [comment]
Fixed in v2.0.10
